download/download.go

```diff
@@ -73,6 +76,12 @@ func (d *Downloader) WithCallback(f func(context.Context, Update)) *Downloader {
 // log messages emitted by the downloader.
 func (d *Downloader) WithLogAttrs(attrs [][2]string) *Downloader {
 	d.logAttrs = attrs
```

Comment:
Do we still need to keep this `logAttrs` field in the `Downloader` struct then...?

Reply:
Looks like you're right. Additionally, I don't find any callers of `WithLogAttrs` so I assume this is unused. Since it's exported though I'll keep it but will remove `logAttrs` from the struct.